--- compiler/systems/t_linux.pas	2020-01-26 12:38:49.026082370 +0100
+++ compiler/systems/t_linux.pas	2020-01-28 02:28:20.365389393 +0100
@@ -141,10 +141,17 @@
       if target_info.abi<>abi_powerpc_elfv2 then
         LibrarySearchPath.AddLibraryPath(sysrootpath,'=/lib64;=/usr/lib64;=/usr/X11R6/lib64',true)
       else
         LibrarySearchPath.AddLibraryPath(sysrootpath,'=/lib64;=/usr/lib/powerpc64le-linux-gnu;=/usr/X11R6/powerpc64le-linux-gnu',true);
-{$else powerpc64}
+{$else}
+{$ifdef aarch64}
+      LibrarySearchPath.AddLibraryPath(sysrootpath,'=/usr/lib',true);
+      LibrarySearchPath.AddLibraryPath(sysrootpath,'=/lib',true);
+      LibrarySearchPath.AddLibraryPath(sysrootpath,'=/usr/lib64',true);
+      LibrarySearchPath.AddLibraryPath(sysrootpath,'=/lib64',true);
+{$else}
       LibrarySearchPath.AddLibraryPath(sysrootpath,'=/lib;=/usr/lib;=/usr/X11R6/lib',true);
+{$endif aarch64}
 {$endif powerpc64}
 {$endif x86_64}
 
 {$ifdef arm}
@@ -222,9 +229,9 @@
 {$endif FPC_ARMHF}
 {$endif arm}
 
 {$ifdef aarch64}
-const defdynlinker='/lib/ld-linux-aarch64.so.1';
+const defdynlinker='/lib64/ld-linux-aarch64.so.1';
 {$endif aarch64}
 
 {$ifdef mips}
   const defdynlinker='/lib/ld.so.1';
